src/Foundation/Features/Shared/Foundation/Header/_Header.cshtml

```diff
@@ -32,7 +32,7 @@ else
     </div>
 }
 
-<header class="header">
+<header class="header sticky-top">
```

Comment:
The requirement is: Add a setting to the header section of the home page, "Enable sticky header", type checkbox, default checked. Not hard coded.